apps/system/js/nfc_handover_manager.js

```diff
@@ -294,31 +294,94 @@ var NfcHandoverManager = {
   },
 
   /**
-   * Performs bluetooth pairing with other device
-   * @param {string} mac MAC address of the peer device
+   * Look for a paired device and invoke the appropriate callback function.
+   * @param {string} mac MAC address of the device
+   * @param {function} foundCb Found callback
+   * @param {function} notFoundCb Not found callback
    * @memberof NfcHandoverManager.prototype
    */
-  _doPairing: function _doPairing(mac) {
-    this._debug('doPairing: ' + mac);
+  _findPairedDevice: function _findPairedDevice(mac, foundCb, notFoundCb) {
```

Comment:
nit: it's better to use promise pattern in the future.
usage => this._findPairedDevice(mac).then(function() {}).catch(function() {});